pyproject.toml

```diff
@@ -104,6 +105,48 @@ write_to = "simtools/_version.py"
 [tool.black]
 line-length = 100
 
+[tool.ruff]
+line-length = 100
+indent-width = 4
+exclude = [
+  "__init__.py",
+  "pyproject.toml",
+  "simtools/_version.py",
+]
+
+format.indent-style = "space"
+format.quote-style = "double"
+format.line-ending = "auto"
+# no documentation linting for test files
```

Comment:
I don't see exactly in which line we ignore the test's repository. Can you please show me which option refers to it?

Reply:
With 'test's repository' you mean the ./tests directory? We don't ignore it.

Reply:
Ok, I mixed up with the test files.